src/ILCompiler.WebAssembly/src/CodeGen/DependencyNodeIterator.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using ILCompiler.DependencyAnalysis;
+using ILCompiler.DependencyAnalysisFramework;
+using Internal.TypeSystem;
+using System.Collections.Generic;
+
+namespace ILCompiler.Compiler.CppCodeGen
+{
+    public class DependencyNodeIterator
```

Comment:
Does webassembly use this class at all, or was this just copypasted with the rest of the CppCodegen stuff?

Reply:
@davidwrighton, will we need this at some point?

Reply:
No, I don't think so.

Reply:
Removed.